src/icon/internal.tsx

```diff
@@ -103,12 +103,32 @@ const InternalIcon = ({
 
   const validIcon = name && Object.prototype.hasOwnProperty.call(icons, name);
 
+  function iconMap(name: IconProps.Name) {
+    const genAiFilledIcon = `<svg
+        width="12"
+        height="12"
+        viewBox="0 0 16 16"
+        xmlns="http://www.w3.org/2000/svg"
+        focusable="false"
+        aria-hidden="true"
+        data-testid="gen-ai-filled"
```

Comment:
What is this data-testid for? We support data attributes on our components. People could just add that if necessary.

Reply:
It is used for the accompanying unit tests.

Reply:
It seems a little strange to me that we use this on our own components, because then this also is rendered in our customers' applications. But doing a little code search shows me that we've done this in a few places in the system already, which I didn't realize. I guess it's fine then